types/fg-loadcss/fg-loadcss-tests.ts

```diff
@@ -0,0 +1,10 @@
+import { loadCSS, onloadCSS } from "fg-loadcss";
```

Comment:
It cannot work that way, the 'onloadCSS' IMO is not a part of a module, can you dlb check this?